.github/workflows/release.yml

```diff
@@ -377,14 +377,23 @@ jobs:
     - name: Install dependencies
       run: dotnet restore
 
+    - uses: dotnet/nbgv@master
+      with:
+        setCommonVars: true
+
     - name: Build
-      run: dotnet build --configuration=LinuxRelease
+      run: |
+        dotnet build --configuration=LinuxRelease -o payload
+        ./src/linux/Packaging.Linux/pack-tar.sh --version=$GitBuildVersionSimple --configuration=Release
+        ./src/linux/Packaging.Linux/pack-deb.sh --version=$GitBuildVersionSimple --configuration=Release
```

Comment:
Just to clarify, on a local dev build we are no longer packing the Debian package or tarball (since we've moved them to scripts that are not in the MSBuild call chain; the `pack-X.sh` scripts must be called manually)?

Reply:
This is true. Probably need to consolidate them for local development - I can make that update.

Reply:
Ended up moving this into its own directory beneath `shared` since it's now cross-platform - just lmk if that needs to change!